locations/spiders/mcdonalds_bg.py

```diff
@@ -1,40 +1,53 @@
-import re
+import json
 
-import scrapy
+from scrapy import FormRequest, Spider
 
-from locations.items import Feature
+from locations.categories import Extras, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
 from locations.spiders.mcdonalds import McDonaldsSpider
 
 
-class McDonaldsBGSpider(scrapy.Spider):
+class McDonaldsBGSpider(Spider):
     name = "mcdonalds_bg"
     item_attributes = McDonaldsSpider.item_attributes
     allowed_domains = ["mcdonalds.bg"]
-    start_urls = ("http://mcdonalds.bg/map/",)
+    start_urls = ["https://mcdonalds.bg/restaurants/"]
 
     def parse(self, response):
-        stores = response.css(".restaurant-info")
-        for store in stores:
-            ref = store.xpath(".//@data-post_id").extract_first()
-            city = store.xpath(".//@data-city_title").extract_first()
-            lat = store.xpath(".//@data-latitude").extract_first()
-            lon = store.xpath(".//@data-longitude").extract_first()
-            address = store.xpath(".//p[1]/text()").extract_first()
-
-            phone = store.xpath(".//p[2]/text()").extract_first()
-            if phone:
-                match = re.search(r"Тел.(.*)", phone)
-                if match:
-                    phone = match.groups()[0].strip()
-
-            properties = {
-                "ref": ref,
-                "phone": phone if phone else "",
-                "lon": lon,
-                "lat": lat,
-                "city": city,
-                "name": "McDonalds",
-                "addr_full": address,
-            }
-
-            yield Feature(**properties)
+        name_ids = response.xpath("//strong[@class='restaurant-item__title']/@data-restaurant-id").extract()
+
+        for id in name_ids:
+            yield FormRequest(
+                url="https://mcdonalds.bg/wp-admin/admin-ajax.php",
+                formdata={"action": "get_restaurant", "restaurant_id": id},
+                callback=self.parse_restaurant,
+                method="POST",
+            )
+
+    def parse_restaurant(self, response):
+        location = json.loads(response.text)["data"]["data"]
+        item = DictParser.parse(location)
+        item["country"] = "bg"
```

Comment:
Maybe this line could be removed as pipeline will assign country from spider name.